### PR TITLE
Use __ptrace_request only on GNU libc

### PR DESCRIPTION
**Pull request by ptrcnull**

### CODE REVIEW

File: ptrace.c [hunk 1]
```diff
@@ -36,6 +36,12 @@
 
 #include "ptrace.h"
 
+#if defined(__GLIBC__)
+#  define PTRACE_REQUEST_TYPE enum __ptrace_request
+#else
```
Comment (jerome-pouiller) [inline ptrace.c:41]: If the host is not using glibc, so which libc is in use?

Comment (ptrcnull) [reply ptrace.c:41]: musl libc, uClibc, anything that's not specifically gnu libc

**Comment by mitar** [comment]
Please merge this.

**Comment by jerome-pouiller** [comment]
I have fixed the comment and applied this fix.

**Comment by mitar** [comment]
Thanks!

**Pull request closed without merging**